cli.js

```diff
@@ -2,14 +2,17 @@
 'use strict';
 var path = require('path')
 
-var windows = process.platform === 'win32'
+    var windows = process.platform === 'win32'
```

Comment:
I think my / our company Formater reformarted a lot of stuff i will change it back.